<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAgenda</class>
 <widget class="QWidget" name="AddAgenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add agenda...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="4">
      <widget class="QLabel" name="agndAlarmMinuteLabel">
       <property name="text">
        <string>minutes ahead.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="agndSubjectLabel">
       <property name="text">
        <string>subject</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QTimeEdit" name="agndFromTimeInput"/>
     </item>
     <item row="1" column="4">
      <widget class="QTimeEdit" name="agndToTimeInput"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="agndFromLabel">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="agndDetailLabel">
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="agndToDtTimLabel">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="agndPosLabel">
       <property name="text">
        <string>At</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="agndAlarmInput"/>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="agndAlarmEnableCheck">
       <property name="text">
        <string>enable</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QTextEdit" name="agndDetailInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>You are the apple of my eye.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="agndAlarmLabel">
       <property name="text">
        <string>alarm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QLineEdit" name="agndSubjectInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Beat bean</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QLineEdit" name="agndPosInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>No.3 Zhongshanlu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QDateEdit" name="agndToDateInput"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QDateEdit" name="agndFromDateInput">
       <property name="displayFormat">
        <string>yyyy-MM-dd dddd</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="agndCancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="agndSubmitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
